src/main/java/io/parallec/core/commander/workflow/ssh/SshProvider.java

```diff
@@ -107,36 +112,49 @@ public ResponseOnSingeRequest executeSshCommand() {
     }
 
     /**
-     * Start ssh session and obtain session.
```

Comment:
Have some concerns on `sshJumpHostMeta` being type `SshMeta`.  SshMeta has around 10 attributes and I believe we just use username port and password , and this recursive pattern looks confusing to people.  

Could we just add the attribute directly to SshMeta, or create a new class just to wrap the attributes used?